db/seeds/prisons/UKI-usk.yml

```diff
@@ -30,10 +30,9 @@ recurring:
   sun:
   - 1400-1600
 unbookable:
-- 2018-12-25
-- 2019-01-01
+- 2019-12-20
+- 2019-12-25
+- 2019-12-26
+- 2020-01-01
 anomalous:
```

Comment:
@Ed-Bajo - if the anomalous section is blank then you need to delete it entirely as otherwise the system assumes you wanted to add something here and it fails.